docs/cep/reference/extensions/io/sse.md

```diff
@@ -0,0 +1,64 @@
+---
+title: SSE
+---
+
+This an extension that receives and publishes events from SSE server
+
+
+## SSE Sink
+
+HTTP SSE sink sends events to all subscribers.
+
+### Syntax:
+
+```js
+CREATE SINK <name> WITH (type="sse-server", server.port="<INT>", server.path="<STRING>", worker.count="<INT>", headers="<STRING>");
+```
+
+### QUERY PARAMETERS:
+
+| Name         | Description |	Default Value |	Possible Data Types	| Optional | Dynamic |
+|--------------|-------------|----------------|---------------------| -------- |---------|
+| server.port  | The listening port of the SSE server which clients need to connect to receive events | | INT	| No | No |
+| server.path  | The listening path of the SSE server which clients need to connect to receive events | | STRING | Yes | No |
+| worker.count | The number of active worker threads to serve the incoming events                     | 1 | INT | Yes | No |
+| headers      | HTTP request headers in format "<key>:<value>,<key>:<value>" | | STRING | Yes | No |
+
+
+
+### EXAMPLE:
+
+```js
+CREATE SINK  PublishStream WITH (type='sse-server', map.type="json", server.port='8020', server.path='testsse') (param1 string);
+```
+
+
+
+
+## SSE Source
+
+HTTP SSE source send a request to a given url and listen to the response stream.
+
+
+### Syntax:
```

Comment:
```suggestion
### SSE Source Syntax
```

Reply:
Hey, Noob question. Why is the source after the sink? I thought the source was the origin of the data and the sink was the destination, so wouldn't source come before sink? I think that's how our stream workers are formatted, anyway.

If so, please make sure that all extensions and such match the way they would be put into an actual stream worker.

Reply:
In case of SSE Source will come after Sink

Reply:
SSE is 17.11, right? We'll merge that info into a different branch. The main branch is only for current functionality.